### ISSUE TITLE
mixed.badssl.com: Don't redirect image to HTTPS.

### ISSUE THREAD

**Issue by lgarron** [lgarron/badssl.com#63]
Chrome treats it as mixed content if any HTTP URL was touched, but it might be more useful for testing other clients if we actually received the content over HTTP.

**Comment by rugk** [comment]
Yes, that's true...
![mixedbaddsslredirect](https://cloud.githubusercontent.com/assets/11966684/10610818/cb8c8336-7749-11e5-85be-ff6ae7e95bc8.PNG)

**Comment by rugk** [comment]
Can't this just be solved by editing/removing [this line](https://github.com/lgarron/badssl.com/blob/master/domains/mixed/mixed.badssl.com.conf#L5) which explicitly specifies the (unwanted) redirect?

**Comment by lgarron** [comment]
Yes, but I want <http://mixed.badssl.com> to redirect to HTTPS. The correct solution would be a few more lines, where we serve only the image in question from HTTP (and also redirect the image from HTTPS to HTTP, just to be sure).
